ui/src/importer/importerTaskDetails/ImporterTaskDetailsComponents/ImportStats.js

```diff
@@ -7,8 +7,9 @@ export const RenderStatistics = ({
   selectedResult,
   applyFilter,
 }) => {
+  const size = statistics.records.value.length > 8 ? 'small' : '';
```

Comment:
what was the problem that this line solves?

Reply:
The statistics component in react-semantic-ui divides the space equally between each statistics, giving each of them a fixed space. With the current implementation, if the number that appears in any statistic has more than 8 characters, it overflows the space, causing it to appear in multiple lines. Making the statistics a bit smaller if that happens prevents this visual issue. The last two screenshots highlight this smaller statistics with large number of records.

Reply:
more than 8 characters, meaning you expect the import's records number to be more than 10 000 000?